programs/uxd/src/state/controller.rs

```diff
@@ -94,6 +97,11 @@ pub struct Controller {
     pub mercurial_vault_depository: Pubkey,
     pub credix_lp_depository: Pubkey,
 
+    // Redeem limitation flags
+    pub limit_outflow_amount_per_day: u64, // or limit_outflow_bps_per_day
```

Comment:
If by `limit_outflow_bps_per_day` you mean that it become something proportionnal to the current total circulating supply then yes I think it needs to be (like 5% or smthg). Then come the zeno paradox when the protocol is fully redeemed though... but minor I think? And we can put a hardcodecd `max(10k or proportionnal value)`

Reply:
Yeah for sure king, agreed, I think I will provide support for both 👌 to avoid the infinite redeem tail